### ISSUE TITLE
Notice: Undefined variable: storeCode

### ISSUE THREAD

**Issue by lykhouzov** [doofinder/doofinder-magento#74]
version 1.8.1 has issue in the helper:
```
<pre>Notice: Undefined variable: storeCode  in /var/github/mageeu_stage/webroot/app/code/community/Doofinder/Feed/Helper/Data.php on line 414

#0 /var/github/mageeu_stage/webroot/app/code/community/Doofinder/Feed/Helper/Data.php(414): mageCoreErrorHandler(8, 'Undefined varia...', '/var/github/mag...', 414, Array)
#1 /var/github/mageeu_stage/webroot/app/code/community/Doofinder/Feed/Helper/Data.php(397): Doofinder_Feed_Helper_Data->_processXmlName('doofinder-{stor...', 'de', true)
#2 /var/github/mageeu_stage/webroot/app/code/community/Doofinder/Feed/Model/Observers/Schedule.php(103): Doofinder_Feed_Helper_Data->getStoreConfig('de')
#3 /var/github/mageeu_stage/webroot/app/code/community/Doofinder/Feed/controllers/DoofinderFeedFeedController.php(38): Doofinder_Feed_Model_Observers_Schedule->updateProcess('de', true, true, true)
#4 /var/github/mageeu_stage/webroot/app/code/core/Mage/Core/Controller/Varien/Action.php(418): Doofinder_Feed_DoofinderFeedFeedController->generateAction()
#5 /var/github/mageeu_stage/webroot/app/code/core/Mage/Core/Controller/Varien/Router/Standard.php(254): Mage_Core_Controller_Varien_Action->dispatch('generate')
#6 /var/github/mageeu_stage/webroot/app/code/core/Mage/Core/Controller/Varien/Front.php(172): Mage_Core_Controller_Varien_Router_Standard->match(Object(Mage_Core_Controller_Request_Http))
#7 /var/github/mageeu_stage/webroot/app/code/core/Mage/Core/Model/App.php(365): Mage_Core_Controller_Varien_Front->dispatch()
#8 /var/github/mageeu_stage/webroot/app/Mage.php(684): Mage_Core_Model_App->run(Array)
#9 /var/github/mageeu_stage/webroot/index.php(83): Mage::run('', 'store')
#10 {main}</pre>
```
it blocks to use `Start Feed Generation Now  ` button in admin panel.

**Comment by carlosescri** [comment]
@lykhouzov thank you for the report. We're on it.

**Comment by carlosescri** [comment]
Fixed in 1.8.2.
